src/common/xo/index.js

```diff
@@ -766,6 +766,18 @@ export const disableRemote = id => (
   )
 )
 
+export const setRemoteName = ({id, name}) => (
```

Comment:
This is not in line with other methods: should be `editRemote(remote, { name, url })`.